### PR TITLE
translated tutorials/wide.md

### PR DESCRIPTION
**Pull request by whatbeg**
tutorials/wide.md 翻译完成。

### CODE REVIEW

**Comment by whatbeg** [comment]
@leviding 翻译完成。

**Comment by leviding** [comment]
@whatbeg 看下翻译计划的 [译文排版要求](https://github.com/xitu/gold-miner/wiki/%E8%AF%91%E6%96%87%E6%8E%92%E7%89%88%E8%A7%84%E5%88%99%E6%8C%87%E5%8C%97)，格式需要调整下，辛苦啦

**Comment by leviding** [comment]
resolve #16

File: tutorials/wide.md [hunk 1]
```diff
@@ -1,59 +1,38 @@
...
 
-In this tutorial, we will use the tf.estimator API in TensorFlow to solve a
-binary classification problem: Given census data about a person such as age,
-education, marital status, and occupation (the features), we will try to predict
-whether or not the person earns more than 50,000 dollars a year (the target
-label). We will train a **logistic regression** model, and given an individual's
-information our model will output a number between 0 and 1, which can be
-interpreted as the probability that the individual has an annual income of over
-50,000 dollars.
+在本教程中，我们将会使用 TensorFlow 中的 tf.estimator API来解决一个二分类问题：给定人口普查数据，有关一个人的如年龄，教育程度，婚姻状况和职业等信息（特征），我们将尝试预测该人每年是否收入超过 5 万美元（目标标签）。我们将训练一个**逻辑回归**模型，给定一个个体的信息，我们的模型会输出一个0到1之间的数，这个数可以解释成此个体年收入超过 5 万美元的概率。
```
Comment (leviding) [inline tutorials/wide.md]: 空格问题

File: tutorials/wide.md [hunk 2]
```diff
@@ -1,59 +1,38 @@
...
 
-## Setup
+## 快速构建
 
-To try the code for this tutorial:
+可以通过如下步骤尝试本教程的代码:
 
-1.  @{$install$Install TensorFlow} if you haven't already.
+1. @{$install$Install TensorFlow} 如果还没安装。
```
Comment (leviding) [inline tutorials/wide.md]: 如果你还没安装，请看 @{$install$Install TensorFlow}

File: tutorials/wide.md [hunk 3]
```diff
@@ -1,59 +1,38 @@
...
 
-## Reading The Census Data
+## 读取 Census 数据
 
-The dataset we'll be using is the
-[Census Income Dataset](https://archive.ics.uci.edu/ml/datasets/Census+Income).
-We have provided
-[data_download.py](https://github.com/tensorflow/models/tree/master/official/wide_deep/data_download.py)
-which downloads the code and performs some additional cleanup.
+我们将要使用的数据集是[Census Income Dataset](https://archive.ics.uci.edu/ml/datasets/Census+Income)。
```
Comment (leviding) [inline tutorials/wide.md]: 空格

File: tutorials/wide.md [hunk 4]
```diff
@@ -1,59 +1,38 @@
...
+我们提供了[data_download.py](https://github.com/tensorflow/models/tree/master/official/wide_deep/data_download.py)来下载数据并进行一些清理。
```
Comment (leviding) [inline tutorials/wide.md]: 空格

File: tutorials/wide.md [hunk 5]
```diff
@@ -1,59 +1,38 @@
...
 
-Since the task is a binary classification problem, we'll construct a label
-column named "label" whose value is 1 if the income is over 50K, and 0
-otherwise. For reference, see `input_fn` in
-[wide_deep.py](https://github.com/tensorflow/models/tree/master/official/wide_deep/wide_deep.py).
+由于该任务是一个二元分类问题，我们将构造一个名为 “label” 的标签列，如果收入超过 5 万美元，那么它的值为 1，否则为 0。有关参考，请参阅 [wide_deep.py](https://github.com/tensorflow/models/tree/master/official/wide_deep/wide_deep.py)。
```
Comment (leviding) [inline tutorials/wide.md]: 空格

File: tutorials/wide.md [hunk 6]
```diff
@@ -1,59 +1,38 @@
...
 
-*   A column is called **categorical** if its value can only be one of the
-    categories in a finite set. For example, the relationship status of a person
-    (wife, husband, unmarried, etc.) or the education level (high school,
-    college, etc.) are categorical columns.
-*   A column is called **continuous** if its value can be any numerical value in
-    a continuous range. For example, the capital gain of a person (e.g. $14,084)
-    is a continuous column.
+*   如果某列的值只能是有限集合中的某个类别，则称该列为**类别列**。例如，一个人的关系状况（妻子，丈夫，未婚等）或教育水平（高中，大学等）都是类别列。
+*   如果某个列的值可以是连续范围内的任何数值，则称该列为**连续列**。例如，一个人的资本收入（例如 $ 14,084）是一个连续列。
```
Comment (leviding) [inline tutorials/wide.md]: $14,084

File: tutorials/wide.md [hunk 7]
```diff
@@ -86,36 +65,22 @@ Here's a list of columns available in the Census Income dataset:
...
-categorical data, we must represent the data as a `SparseTensor`. This data
-format is good for representing sparse data. Our `input_fn` uses the `tf.data`
-API, which makes it easy to apply transformations to our dataset:
+## 将数据转化为张量
+
+在构建 tf.estimator 模型时，输入数据通过 Input Builder 函数指定。 此构建函数在稍后传递给 tf.estimator.Estimator 方法（如 `train` 和 `evaluate`）之前不会被调用。
+这个函数的目的是构造输入数据，它以 @{tf.Tensor}s 或 @{tf.SparseTensor} 的形式表示。
+更详细地说，输入构建函数将成对返回以下配对：
+
+1.  `features`: 一个由特征列名到 `Tensors` 或 `SparseTensors` 的映射。
```
Comment (leviding) [inline tutorials/wide.md]: 冒号和后面的空格换成中文冒号

File: tutorials/wide.md [hunk 8]
```diff
@@ -86,36 +65,22 @@ Here's a list of columns available in the Census Income dataset:
...
+2.  `labels`: 一个包含标签列的 `Tensor`。
```
Comment (leviding) [inline tutorials/wide.md]: 冒号和后面的空格换成中文冒号

File: tutorials/wide.md [hunk 9]
```diff
@@ -86,36 +65,22 @@ Here's a list of columns available in the Census Income dataset:
...
+
+`features` 的键将用于构建下一节中提到的列。由于我们想在调用 `train` 和 `evaluate` 方法时使用不同的数据，所以我们定义一个方法，它根据给定的数据返回一个输入函数。 请注意，返回的输入函数将在构建 TensorFlow 计算图时调用，而不是在运行计算图时调用。它返回的是输入数据作为 `Tensor`（或 `SparseTensor`）的表示，即TensorFlow计算的基本单位。
```
Comment (leviding) [inline tutorials/wide.md]: 空格

File: tutorials/wide.md [hunk 10]
```diff
@@ -145,25 +109,14 @@ def input_fn(data_file, num_epochs, shuffle, batch_size):
...
+### 基本的类别型特征列
 
-To define a feature column for a categorical feature, we can create a
-`CategoricalColumn` using the tf.feature_column API. If you know the set of all
-possible feature values of a column and there are only a few of them, you can
-use `categorical_column_with_vocabulary_list`. Each key in the list will get
-assigned an auto-incremental ID starting from 0. For example, for the
-`relationship` column we can assign the feature string "Husband" to an integer
-ID of 0 and "Not-in-family" to 1, etc., by doing:
+要为类别型特征定义特征列，我们可以使用 tf.feature_column API 创建一个 `CategoricalColumn`。如果你知道一列中所有可能的特征值的集合，并且只有少数几个，你可以使用 `categorical_column_with_vocabulary_list`。 列表中的每个键都将被分配一个从 0 开始的自动增长的 ID。例如，对于 `relationship` 列，通过以下操作，我们可以为特征字符串 “Husband” 分配给一个整数 0 作为 ID ，为 “Not-in-family” 设置 1 作为 ID 等等：
```
Comment (leviding) [inline tutorials/wide.md]: 列表中的每个键都将被分配一个从 前面多了个空格

File: tutorials/wide.md [hunk 11]
```diff
@@ -194,15 +144,10 @@ ID  | Feature
...
 
-No matter which way we choose to define a `SparseColumn`, each feature string
-will be mapped into an integer ID by looking up a fixed mapping or by hashing.
-Note that hashing collisions are possible, but may not significantly impact the
-model quality. Under the hood, the `LinearModel` class is responsible for
-managing the mapping and creating `tf.Variable` to store the model parameters
-(also known as model weights) for each feature ID. The model parameters will be
-learned through the model training process we'll go through later.
+无论我们选择哪种方式定义 `SparseColumn`，通过查找固定的映射或散列，每个特征字符串都将被映射为一个整数 ID。
+请注意，哈希碰撞是可能的，但可能不会显着影响模型质量。 在底层实现上，`LinearModel` 类负责管理映射并创建 `tf.Variable` 来存储每个特征ID的模型参数（也称为模型权重）。模型参数将通过后面将要讨论的模型训练过程来学习。
```
Comment (leviding) [inline tutorials/wide.md]: 空格

File: tutorials/wide.md [hunk 12]
```diff
@@ -244,76 +188,51 @@ capital_loss = tf.feature_column.numeric_column('capital_loss')
...
-early stage of one's career, then the growth may slow at some point, and finally
-the income decreases after retirement. In this scenario, using the raw `age` as
-a real-valued feature column might not be a good choice because the model can
-only learn one of the three cases:
+有时连续特征和标签之间的关系不是线性的。 假设这么一个例子，一个人的收入可能会在职业生涯的早期阶段随着年龄的增长而增长，然后增长可能会放缓，最终退休后的收入会减少。 在这种情况下，使用原始的 `age` 作为实值特征列可能不是一个好选择，因为模型只能学习三种情况之一：
 
-1.  Income always increases at some rate as age grows (positive correlation),
-1.  Income always decreases at some rate as age grows (negative correlation), or
-1.  Income stays the same no matter at what age (no correlation)
+1.  随着年龄增长，收入总是以某种速度增加（正相关），
```
Comment (leviding) [inline tutorials/wide.md]: 这三行优化一下

File: tutorials/wide.md [hunk 13]
```diff
@@ -332,23 +251,17 @@ model = tf.estimator.LinearClassifier(
     model_dir=model_dir, feature_columns=base_columns + crossed_columns)
 ```
 
-The model also automatically learns a bias term, which controls the prediction
-one would make without observing any features (see the section "How Logistic
-Regression Works" for more explanations). The learned model files will be stored
-in `model_dir`.
+该模型还会自动学习一个偏差项，它可以控制在不观察任何特征的情况下进行的预测（有关更多解释，请参见“逻辑回归的工作原理”一节）。 学习的模型文件将被存储在`model_dir` 中。
```
Comment (leviding) [inline tutorials/wide.md]: 空格

File: tutorials/wide.md [hunk 14]
```diff
@@ -332,23 +251,17 @@ model = tf.estimator.LinearClassifier(
...
 
-## Training and Evaluating Our Model
+## 训练和评估我们的模型
 
-After adding all the features to the model, now let's look at how to actually
-train the model. Training a model is just a single command using the
-tf.estimator API:
+将所有特征添加到模型后，现在让我们看看如何实际训练模型。使用tf.estimator API 训练模型仅需用一个命令：
```
Comment (leviding) [inline tutorials/wide.md]: 空格

File: tutorials/wide.md [hunk 15]
```diff
@@ -387,65 +289,26 @@ model = tf.estimator.LinearClassifier(
...
-## How Logistic Regression Works
-
-Finally, let's take a minute to talk about what the Logistic Regression model
-actually looks like in case you're not already familiar with it. We'll denote
-the label as \\(Y\\), and the set of observed features as a feature vector
-\\(\mathbf{x}=[x_1, x_2, ..., x_d]\\). We define \\(Y=1\\) if an individual
-earned > 50,000 dollars and \\(Y=0\\) otherwise. In Logistic Regression, the
-probability of the label being positive (\\(Y=1\\)) given the features
-\\(\mathbf{x}\\) is given as:
+L1 和 L2 正则化之间的一个重要区别是，L1 正则化倾向于使模型权重保持为零，从而创建更稀疏的模型，而 L2 正则化也尝试使模型权重接近于零，但不一定为零。 因此，如果增加 L1 正则化的强度，你将获得一个更小的模型，因为模型的许多权重将为零。如果特征空间非常大但稀疏，并且存在资源限制，从而无法训练和上线太大的模型时，这通常是可取的。
```
Comment (leviding) [inline tutorials/wide.md]: 空格

**Comment by whatbeg** [comment]
@leviding 多谢校对，修正了大量的格式问题。

**Comment by pkuwwt** [comment]
我把表格里的英文翻译了一下，下次注意一下。@whatbeg

**Comment by leviding** [comment]
@pkuwwt 这种 @{$install$Install Tensorflow} 里面的内容先不翻译，保留原样。这是用于文档间链接的标记，现在 web 端已经做得差不多了，需要这个进行文档间的互联。

**Comment by leviding** [comment]
如果还没安装，请 @{$install$Install TensorFlow}。 这个地方语序是需要注意的， @pkuwwt 改的很赞
